src/coreclr/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -288,7 +290,20 @@
     <Compile Include="$(CommonPath)Interop\Windows\OleAut32\Interop.VariantClear.cs">
       <Link>Common\Interop\Windows\OleAut32\Interop.VariantClear.cs</Link>
     </Compile>
+    <Compile Include="$(BclSourcesRoot)\System\Threading\PreAllocatedOverlapped.CoreCLR.Windows.cs" />
+    <Compile Include="$(BclSourcesRoot)\System\Threading\RegisteredWaitHandle.CoreCLR.Windows.cs" />
     <Compile Include="$(BclSourcesRoot)\System\Threading\ThreadPool.CoreCLR.Windows.cs" />
+    <Compile Include="$(BclSourcesRoot)\System\Threading\ThreadPoolBoundHandle.CoreCLR.Windows.cs" />
```

Comment:
Could you please move everything under `src\libraries`? None of this should be CoreCLR specific.

Reply:
(Also move the Win32 threadpool files from under src\coreclr\nativeaot under libraries.)